regression/cbmc-java-concurrency/several_threads/A.java

```diff
@@ -0,0 +1,97 @@
+import java.lang.Thread;
+import org.cprover.CProver;
+
+public class A
+{
+
+  // expected verification success
+  public void me()
+  {
+    B b= new B();
+    b.me();
+  }
+
+  // expected verification failure
+  public void me2()
+  {
+    C c= new C();
+    c.me();
+  }
+
+  // Create 32 Threads, no shared variables
```

Comment:
- modify test to have 2 threads
- modify comment accordinly
- write FIXME saying that this shoud pass with 32 threads in reasonable time